src/main.rs

```diff
@@ -285,29 +283,32 @@ struct ReplayArgs {
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let opt = Cli::parse();
-
     let log_level_filter = LevelFilter::from(opt.log);
-    let log_config = ConfigBuilder::new()
-        .add_filter_allow_str("era_test_node")
-        .build();
-    CombinedLogger::init(vec![
-        TermLogger::new(
-            log_level_filter,
-            log_config.clone(),
-            TerminalMode::Mixed,
-            ColorChoice::Auto,
-        ),
-        WriteLogger::new(
-            log_level_filter,
-            log_config,
-            File::create(opt.log_file_path).unwrap(),
-        ),
-    ])
-    .expect("failed instantiating logger");
+    let log_file = File::create(opt.log_file_path)?;
+
+    // Initialize the tracing subscriber
+    tracing_subscriber::registry()
+        .with(
+            EnvFilter::from_default_env()
```

Comment:
@nbaztec Could we explore using a mutable filter here? I would LOVE to be able to update the LogLevelFilter without restarting the node and just use an API, ex: `config_setLogLevelFilter("trace")`

```rust
// Rough pseudo code
let (filter, handle) = reload::Layer::new(EnvFilter::from_default_env().add_directive(...));

tracing_subscriber::registry().with(filter).init();

// ... then in the API call
let new_filter = EnvFilter::from_default_env().add_directive(format!("era_test_node={}", new_log_level_filter));
handle.reload(new_filter).unwrap();

```